### DISCUSSION TITLE
Utilizing existing Azure SQL Database

### DISCUSSION THREAD

**Discussion by JohnMarsing** [oqtane/oqtane.framework#1487] [Q&A]
I'm thinking about porting an existing Blazor web app that has a Azure SQL database and I'm wondering if I can merge that with the database used by Octane. I'm trying to avoid paying for two databases.
 Thanks.

**Comment by sbwalker** [comment]
It should be possible as long as your existing database does not have any tables whose names conflict with Oqtane's core tables.

**Reply by JohnMarsing** [reply]
Thanks, I'll give it a try
